app/src/main/res/layout/fragment_profile.xml

```diff
@@ -0,0 +1,151 @@
+                    android:layout_height="wrap_content"
+                    android:textSize="@dimen/fontHeading4"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    tools:text="Welcome, Mohab Elziny" />
+
+                <TextView
+                    android:id="@+id/textView9"
+                    style="@style/fontBoldStyle"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="4dp"
+                    android:gravity="center"
+                    android:minWidth="48dp"
+                    android:minHeight="48dp"
+                    android:text="@string/orders"
+                    android:textSize="@dimen/fontHeading4"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/txt_welcome_message" />
+
+                <TextView
+                    android:id="@+id/btn_more_orders"
+                    style="@style/fontBoldStyle"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:clickable="true"
+                    android:focusable="true"
+                    android:gravity="center"
+                    android:minWidth="48dp"
+                    android:minHeight="48dp"
+                    android:text="@string/more"
+                    android:textColor="@color/Primary"
+                    android:textSize="@dimen/fontHeading4"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintTop_toTopOf="@+id/textView9" />
+
+                <androidx.recyclerview.widget.RecyclerView
+                    android:id="@+id/orders_recycler_view"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="4dp"
+                    android:isScrollContainer="false"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/textView9"
+                    tools:itemCount="2" />
+
+                <TextView
+                    android:id="@+id/textView10"
+                    style="@style/fontBoldStyle"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="4dp"
+                    android:gravity="center"
+                    android:minWidth="48dp"
+                    android:minHeight="48dp"
+                    android:text="@string/my_favorites"
+                    android:textSize="@dimen/fontHeading4"
+                    app:layout_constraintStart_toStartOf="parent"
```

Comment:
it is beter to use name of fragment in id to know that tis view belongs to this fragment